src/channel/fixed_size_payment.rs

```diff
@@ -75,7 +70,7 @@ impl<const P: usize> From<Params<P>> for perunwire::Params {
                 buf
             },
             parts: value.participants.map(|a| a.0.to_vec()).to_vec(),
-            app: value.app.0.to_vec(),
+            app: [0; 0].to_vec(),
```

Comment:
Just as a note: This can be written as: `app: vec![],` (which I think makes it clearer / easier to see that it creates an empty Vec).

Reply:
Good suggestion. I have updated.